### ISSUE TITLE
Access Request-Headers AFTER hitting the request

### ISSUE THREAD

**Issue by Anuj28282828** [a85/POSTMan-Chrome-Extension#11018]
### Is there an existing request for this feature?

- [X] I have searched the existing issues for this feature request and I know that duplicates will be closed

### Is your feature request related to a problem?

My question:
Is it possible to access request-header after the request is hit?

My pre-request script looks like :

var Cookie = null
pm.sendRequest("http://www.test.com/anuj", function (err, response, options) {
    //console.log(pm.cookies);
    console.log(request)
    Cookie = options.cookies.members[0].value;
    //console.log(options.cookies.members[0].value);
});
pm.globals.set("Cookie", Cookie);
pm.environment.set("Cookie", Cookie);
pm.request.headers.add({key: "Cookie", value: Cookie});

So basically I want to capture the auto-generated(auto-injected) Request-Header Cookie from the pre-request script and use it in the actual request

I’ve already tried:

options.cookies.member
request.headers
pm.cookies

### Describe the solution you'd like

options.cookies.member
request.headers
pm.cookies

**Comment by DannyDainton** [comment]
Hey @Anuj28282828

This isn't a feature request to add additional functionality into the product.

Can I ask that you raise this question over on the Postman community fourm.

https://community.postman.com
